gradle/gradle-mvn-push.gradle

```diff
@@ -107,3 +107,16 @@ afterEvaluate { project ->
         archives javadocJar
     }
 }
+
+task uploadArchivesLocal(type: Upload) {
```

Comment:
have you consider having this outside of this file? this file is from an opensource repo.
might be easier to replace and update when needed. https://github.com/chrisbanes/gradle-mvn-push

Reply:
I had not considered that, and I actually didn't know about that repo 😅 
But it does makes sense, I will separate into a different file and add it as a kind of file decorator.